cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/services/ApplicationLifecycleService.java

```diff
@@ -315,10 +316,30 @@ public ApplicationDetail getAppDetail(ApplicationId appId) throws Exception {
     if (appSpec == null) {
       throw new ApplicationNotFoundException(appId);
     }
+    // TODO: fetch creation time from store
```

Comment:
This should be unblocked now, right? Could add needed columns now

Reply:
yes, that would be the next rev